apps/client/src/utils/usePassPropsToChildren.ts

```diff
@@ -8,13 +8,13 @@ type Callback = (index?: number) => any;
  *
  * Use in case we need to pass props to children.
  * @param propsOrCallback this could be an object or a callback with the index as a parameter
- * @param childrenProps React children
+ * @param childrenProp React children
```

Comment:
Can you change the order of these `@params` too?

Reply:
All fixed https://github.com/Amsterdam/vergunningcheck/pull/440/commits/bdd53468db1949504cda9c5fdc975bec0298e21e